tools/kontaktdaten.py

```diff
@@ -109,27 +189,68 @@ def validate_kontakt(kontakt):
                 f"Ungültiger Key {json.dumps(key)}:\n{str(exc)}")
 
 
-def validate_phone(phone):
+def validate_phone(phone: str):
+    """
+    Validiert Telefonnummer auf: Typ, Präfix, "leer"
+
+    Args:
+        phone (str): Telefonnummer
+
+    Raises:
+        ValidationError: Typ ist nicht str
+        ValidationError: Telefonnummer ungültig
+    """
+
     if not isinstance(phone, str):
         raise ValidationError("Muss eine Zeichenkette sein")
 
-    if not re.match(r"^(\+49)?[0-9]+$", phone):
+    if not re.match(r"^\+49[1-9][0-9]+$", phone):
         raise ValidationError(
-            f"Ungültige Telefonnummer {json.dumps(phone)} - darf nur aus Ziffern bestehen")
+            f"Ungültige Telefonnummer {json.dumps(phone)}")
 
 
-def validate_hausnummer(hausnummer):
+def validate_hausnummer(hausnummer: str):
+    """
+    Validiert Hausnummer auf: Typ, Länge, "leer"
+
+    Args:
+        hausnummer (str): hausnummer
+
+    Raises:
+        ValidationError: Typ ist nicht str
+        ValidationError: Zeichenkette ist länger als 20 Zeichen
+        ValidationError: Zeichenkette ist leer
+    """
+
     if not isinstance(hausnummer, str):
         raise ValidationError("Muss eine Zeichenkette sein")
 
     if len(hausnummer) > 20:
         raise ValidationError(
             f"Hausnummer {json.dumps(hausnummer)} ist zu lang - maximal 20 Zeichen erlaubt")
 
+    if hausnummer.strip() == "":
+        raise ValidationError("Hausnummer ist leer")
+
+
+def validate_email(email: str):
+    """
+    Validiert eMail auf: Typ, gültigkeit, "leer"
+
+    Args:
+        email (str): email
+
+    Raises:
+        ValidationError: Typ ist nicht str
+        ValidationError: Zeichenkette enhält kein @
+        ValidationError: Zeichenkette ist leer
+    """
 
-def validate_email(email):
     if not isinstance(email, str):
         raise ValidationError("Muss eine Zeichenkette sein")
 
     if '@' not in parseaddr(email)[1]:
         raise ValidationError(f"Ungültige E-Mail-Adresse {json.dumps(email)}")
+
+    if email.strip() == "":
+        raise ValidationError("Mail ist leer")
```

Comment:
https://www.duden.de/rechtschreibung/E_Mail_Adresse

```suggestion
        raise ValidationError("E-Mail-Adresse ist leer")
```